pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -253,16 +266,21 @@ func (r *reconcilerBase) createOrPatchDeployment(ctx context.Context, declared *
 // This function aims to address the fight among Autopilot, VPA and the reconciler since they all update the resources.
 // Below is the resolution:
 // 1. If VPA is enabled (no matter if it is a standard cluster or an Autopilot cluster),
-//    use the current resources and ignore the override defined in the declared Deployment
-//    because VPA will scale up and down automatically.
+//
+//	use the current resources and ignore the override defined in the declared Deployment
+//	because VPA will scale up and down automatically.
```

Comment:
It's not VPA specific. We can remove L271.

Reply:
comments have been updated to a more clear description